### PR TITLE
Adding guide for Commerce Custom Events

### PR DESCRIPTION
**Pull request by fe-lix-**
Guide on how to integrate Adobe Commerce directly with Adobe I/O Events

## Description

This guide go through the creation of : Event Provider, Event publishing, Event Registration and Reception in the context of Adobe Commerce and Adobe I/O Events.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/using/commerce-custom-events.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+- Select `Create new project`
+
+  ![Create new project](../img/AdobeCommerceAPIStep1.png "Create new project")
+
+- Select `Add API`
+
+  ![Add API](../img/AdobeCommerceAPIStep2.png "Add API")
+
+- Filter by `Adobe Services`
+- Select `I/O Management API`
```
Comment (vkublytskyi) [inline src/pages/guides/using/commerce-custom-events.md:42]: Should be `I/O Events` as per the screenshot

File: src/pages/guides/using/commerce-custom-events.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+These instruction describe how to set up and get started with Adobe I/O Events for Adobe Commerce. You can use Adobe I/O for sending Adobe Commerce user driven events.
+
+## Introduction
+
+This document will describe all steps necessary to setup, emit and receive events. Firstly, the event provdire will be set up, then we will trigger events from Adobe Commerce and finally an event registration will be created.
+
+## Requirements
+
+You will need for this tutorial:
+- an Adobe Commerce instance
```
Comment (vkublytskyi) [inline src/pages/guides/using/commerce-custom-events.md:16]: In fact, the guide assumes Adobe Commerce instance in Adobe Commerce Cloud. We should highlight these details or change examples to use the local Magento instance.

File: src/pages/guides/using/commerce-custom-events.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+
+These instruction describe how to set up and get started with Adobe I/O Events for Adobe Commerce. You can use Adobe I/O for sending Adobe Commerce user driven events.
+
+## Introduction
+
+This document will describe all steps necessary to setup, emit and receive events. Firstly, the event provdire will be set up, then we will trigger events from Adobe Commerce and finally an event registration will be created.
+
+## Requirements
+
+You will need for this tutorial:
```
Comment (vkublytskyi) [inline src/pages/guides/using/commerce-custom-events.md:15]: `magento-cloud` CLI tool is also needed. We should mention it and provide a link to https://devdocs.magento.com/cloud/reference/cli-ref-topic.html

File: src/pages/guides/using/commerce-custom-events.md [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+*Commands to configure*
+
+    magento-cloud variable:set env:ADOBE_IO_CONSOLE_CONFIGURATION "`cat 691FuchsiaReptile-343284-Stage.json`"
+    magento-cloud variable:set env:ADOBE_IO_TECHACCT_PRIVATE_KEY "`cat config/private.key`"
+    magento-cloud variable:set env:ADOBE_IO_EVENTS_INSTANCE_ID "commerce-343284-instance-stage"
+
+#### Install the Adobe I/O Event module
+
+In your Adobe Commerce instead run `composer require magento/module-adobe-io-events`
```
Comment (OneCricketeer) [inline src/pages/guides/using/commerce-custom-events.md:82]: `s/instead/instance/`

File: src/pages/guides/using/commerce-custom-events.md [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+From the previous step will we require the following files:
+- the workspace configuration, ours is `691FuchsiaReptile-343284-Stage.json` yours will match your Runtime namespace name.
+- the private key from your service account, ours is `private.key`
+
+#### Configure Adobe Commerce
+
+You will need to create three environment variables containing:
+- Adobe Console Workspace Configuration
+- Technical Account Private Key
+- Intance ID
```
Comment (OneCricketeer) [inline src/pages/guides/using/commerce-custom-events.md:72]: Instance

File: src/pages/guides/using/commerce-custom-events.md [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+
+- Select `Add Service` and the `Event`
+
+  ![Add Event](../img/AdobeCommerceEventRegistrationStep004.png "Add Event")
+
+- Scroll until you find your event provider `My Commerce Instance`
+
+  ![Select Event Provider](../img/AdobeCommerceEventRegistrationStep005.png "Select Event Provider")
+
+- Select both event suscription `Product Created` and `Product Updated`
```
Comment (OneCricketeer) [inline src/pages/guides/using/commerce-custom-events.md:262]: Subscription

**Pull request closed without merging**